test/test.py

```diff
@@ -22,6 +22,8 @@ class TestBase(unittest.TestCase):
     to common functionality.
     """
 
+    maxDiff = None
+
```

Comment:
Did you mean to change this?

Reply:
I did. I also meant to put it in a separate commit. Ugh. I would leave it in, if you don't mind. Without this failing stage tests would not show the pieces that failed in case the diff was too big.